swarm/tools/runs_gc.py

```
@@ -58,32 +58,43 @@ class RunInfo:
     run_type: str  # "active", "example", "legacy"
-    size_bytes: int
     mtime: datetime
     has_meta: bool
     is_corrupt: bool
     tags: List[str]
+    _size_bytes: int | None = None
 
     @property
     def age_days(self) -> float:
         """Age in days since last modification."""
         now = datetime.now(timezone.utc)
```

Ruff (0.15.6)

[warning] Use `datetime.UTC` alias

Convert to `datetime.UTC` alias

(UP017)